test/test_schema.py

```diff
@@ -135,6 +135,13 @@ def test_column_subset_error(self):
         # should raise a PanSchArgumentError
         self.assertRaises(PanSchArgumentError, self.schema.validate, df, columns=['c'])
 
+    def test_column_not_present_shown(self):
+
+        df = pd.DataFrame.from_dict({'a': [1, 2, 3]})
+
+        out = self.schema.validate(df, columns=['a', 'b'])
+        assert out[0].message == 'The column b exists in the schema but not in the data frame'
```

Comment:
This should be a `self.assert()` method (https://docs.python.org/3.7/library/unittest.html#unittest.TestCase.debug)